pkg/setting/setting_plugins.go

```diff
@@ -36,6 +36,55 @@ var (
 	}
 )
 
+// migrateInstallPluginsToPreinstall populates cfg:plugins.preinstall
+// if cfg:plugins.preinstall is empty and GF_INSTALL_PLUGINS is set
+func (cfg *Cfg) migrateInstallPluginsToPreinstall(iniFile *ini.File, installPluginsVal string) {
+	if installPluginsVal == "" {
+		return
+	}
+	pluginsSection := iniFile.Section("plugins")
+	preinstall := pluginsSection.Key("preinstall").MustString("")
+	if preinstall != "" {
```

Comment:
in case there is some preinstall plugins already set, you will need to merge both values (and in case of conflict, give preference to the preinstall setup)